src/wasm-lib/tests/executor/inputs/no_visuals/boolean_logical_or.kcl

```diff
@@ -0,0 +1,32 @@
+let aa = true | false
```

Comment:
You no longer need `let` keyword to declare a variable, it's optional. Style guide is to omit them :)

Reply:
sg, updated!